### ISSUE TITLE
FAQ / Known Issues

### ISSUE THREAD

**Issue by SourMesen** [SourMesen/Mesen-S#4]
As of the ***latest dev builds***:
* These enhancement chips are still unsupported:
  * ST018 (Hayazashi Nidan Morita Shogi 2)
* These games freeze at boot (or soon after):
  * Battle Grand Prix
  * Kishin Douji Zenki - Tenchi Meidou

Other than the games listed above, everything else *should* be working properly. If you find issues with any other game, please let me know!

**Comment by tewtal** [comment]
Tales of Phantasia (and other ExHiROM games/hacks) does not currently boot since the additional data after the first 4MB of ROM is not mapped to banks 40-7D (and mirrored to 00-3F).

**Comment by SourMesen** [comment]
@tewtal Thanks - I had (wrongly) assumed that ToP used one of the enhancement chips, so I hadn't even taken a look at exhirom carts yet.  I just added support for them, but it looks like ToP has sound-related issues.  The other ExHiROM game seems to be working correctly now, though.

**Comment by angrylion7** [comment]
Kirby's Dream Land 3 USA has prolonged pauses between different segments of prerecorded gameplay (in other words, between different levels) in the game's demo mode. These delays are accompanied by the black screen and sound crackle, are unnaturally long and do not match bsnes at all. Tested with this Windows build by simply loading the game and not pressing any keys after that:
https://ci.appveyor.com/project/Sour/mesen-s/builds/26519814

**Comment by SourMesen** [comment]
@angrylion7  Thanks! This is fixed as of the latest commit

**Comment by ghost** [comment]
wow! very short list of short comings for such a young emulator.


Impressive.

**Comment by ghost** [comment]
very high performance out of the dev build un optimized in a dot render ppu.


I truly love your work on both mesen and mesen-s

**Comment by dukeblooders** [comment]
Super Mario Kart does not work on my computer (Windows 10, x64) with the libretro core and default settings... it stays stuck on the Nintendo logo.
I tried PAL and US versions, .smc and .sfc files. Other games work fine. Any idea ?

**Comment by ghost** [comment]
Sounds like you need dsp1b.rom

**Comment by dukeblooders** [comment]
Uh.. didn't know there was "external" bios files for Snes, but I think I'll wait for a core update (if it's fixed someday). Thanks for your reply.

**Comment by ghost** [comment]
Fixed? Not broken....

**Comment by dukeblooders** [comment]
Yeah, fix is probably not the best word. But I imagine it would be possible to bypass a bios file usage (or embed it into the core).

**Comment by ghost** [comment]
> Yeah, fix is probably not the best word. But I imagine it would be possible to bypass a bios file usage (or embed it into the core).

A simulation less accurate can be embedded. Less desirable. If you want that go for SNES9X

**Comment by keithbowes** [comment]
> Super Mario Kart does not work on my computer (Windows 10, x64) with the libretro core and default settings... it stays stuck on the Nintendo logo.
> I tried PAL and US versions, .smc and .sfc files. Other games work fine. Any idea ?

Me too.  That used to happen in other emulators too (e.g. bsnes and mednafen's snes-faust) before the developers added support for the required BIOS.  I'm guessing it's just because Mesen-S is so early in development.

**Comment by dukeblooders** [comment]
> Me too.  That used to happen in other emulators too (e.g. bsnes and mednafen's snes-faust) before the developers added support for the required BIOS.  I'm guessing it's just because Mesen-S is so early in development.

For libretro, put all Snes bios files in a Firmware folder (in system folder) and it should work.

**Comment by SourMesen** [comment]
I think I changed this a few months ago to make it so the firmware files must be in the "system" folder directly (to match the bsnes core's behavior, iirc.)

Mario Kart and the like are all supported now - if the game is not loading as expected, you are missing firmware files (or they have the wrong name, or are in the wrong folder.)  bsnes/mednafen fall back to using high-level emulation when they cannot find the firmware files - this way of emulating the game is less accurate (and requires a *lot* more code/effort to maintain, so it's not something I am ever planning to support.)

**Comment by pakitovic** [comment]
Hello, @SourMesen is MSU-1 emulation planned? I only use other emulators in retroarch just for it.
Thanks for this great emulator and his brother (Mesen for NES)

**Comment by SourMesen** [comment]
@pakitovic MSU-1 emulation is already available, but it isn't in 0.3.0.  Grab the appveyor dev builds linked in the readme if you want to use MSU-1 patches, until 0.4.0 is released.

**Comment by pakitovic** [comment]
MSU-1 emulation works perfect with dev build. Try out this [Zelda modification](http://www.romhacking.net/hacks/2483/)
It's a pity that additional files could not be separate from de unzipped rom as the HdPacks does.
Again @SourMesen thanks for your impressive work.

**Comment by FateForWindows** [comment]
@SourMesen Rockman X2's earlier prototype fails to initialize the CX4 chip even after the header is fixed as the emulator doesn't identify its use of it, making it completely unplayable. The later prototype doesn't have this issue.

**Comment by CST1229** [comment]
For me, the Game Boy stuff does not show up. At all.
And yes, I'm probably on the latest version, I just downloaded this yesterday off the official website.

![Where's the Game Boy settings?](https://user-images.githubusercontent.com/68464103/91145370-a5010d00-e6b5-11ea-89b4-ea9db722320e.png)


You can also see from the above image that loading a GB ROM, or booting the SGB ROM directly did not work.
Also you can't choose to boot GB files from the load menu.

**Comment by Max833** [comment]
You have to use the development build: https://ci.appveyor.com/project/Sour/mesen-s/build/artifacts

**Comment by CST1229** [comment]
Oh I didn't even know that it is a beta feature, manual didn't even say anything about it being a dev feature
